content/blog.md

```diff
@@ -2,648 +2,186 @@
-			</li>
-		  </ul>
-		</div>
-		<div class="post-title">
-		  <a href="https://gianarb.it/blog/what-is-distributed-tracing-opentracing-opencensus">
-			<h3>What is distributed tracing. Zoom on opencensus and opentracing</h3>
-		  </a>
-		</div>
-		<div class="post-content">
-			<p>A few months ago I started to actively study, support and use opentracing and more in general the distributed tracing topic. In this article, I will share something about…<br /><br /></p>
-		</div>
-	  </div>
-	</div>
-	<!-- BLOG ITEM 3 -->
-	<div class="col-md-4">
-	  <div class="blog-item">
-	    <div class="post-meta">
-		  <ul>
-		    <li>
-			  <i class="fa fa-tags"></i>
-			  <span>OC</span>
-			</li>
-			<li>
-			  <i class="fa fa-calendar-o"></i>
-			  <span>January 23, 2018</span>
-			</li>
-		  </ul>
-		</div>
-		<div class="post-title">
-		  <a href="http://dieswaytoofast.blogspot.com/2018/01/opencensustowards-harmonizing-your.html">
-			<h3>OpenCensus — towards harmonizing your Instrumentation</h3>
-		  </a>
-		</div>
-		<div class="post-content">
-			<p>You’ve really gotten into this whole Observability thing, and have started plugging Prometheus into, well, everything that doesn’t already have it.<br /><br /><br /></p>
-		</div>
-	  </div>
-	</div>
-  </div> <!-- ROW 7 END -->
-</div> <!-- ROW END -->
-<br />
-
-
-<!-- ROW 8 START -->
-<div class="row">   
-  <!-- BLOG CONTENT START -->
-  <div class="blog-c">
-	<!-- BLOG ITEM 1 -->
-  <div class="col-md-4">
-	  <div class="blog-item">
-		<div class="post-meta">
-		  <ul>
-			<li>
-			  <i class="fa fa-tags"></i>
-			  <span>OC</span>
-			</li>
-			<li>
-			  <i class="fa fa-calendar-o"></i>
-			  <span>January 18, 2018</span>
-			</li>
-		  </ul>
-		</div>
-		<div class="post-title">
-		  <a href="https://kausal.co/blog/opencensus-prometheus-kausal/">
-			<h3>OpenCensus with Prometheus and Kubernetes</h3>
-		  </a>
-		</div>
-		<div class="post-content">
-			<p>Yesterday, Google announced OpenCensus, an instrumentation framework for monitoring and tracing. It comes with a set of client libraries for Golang and Java, with more to come.<br /><br /></p>
-		</div>
-	  </div>
-	</div>
-  <!-- BLOG ITEM 2 -->
-	<div class="col-md-4">
-	  <div class="blog-item">
-	    <div class="post-meta">
-		  <ul>
-		    <li>
-			  <i class="fa fa-tags"></i>
-			  <span>OC</span>
-			</li>
-			<li>
-			  <i class="fa fa-calendar-o"></i>
-			  <span>January 17, 2018</span>
-			</li>
-		  </ul>
-		</div>
-		<div class="post-title">
-		  <a href="https://opensource.googleblog.com/2018/01/opencensus.html">
-			<h3>OpenCensus: A Stats Collection and Distributed Tracing Framework</h3>
-		  </a>
-		</div>
-		<div class="post-content">
-			<p>Today we’re pleased to announce the release of OpenCensus, a vendor-neutral open source library for metric collection and tracing.<br /><br /><br /></p>
-		</div>
-	  </div>
-	</div>
-  </div> <!-- ROW 8 END -->
-</div> <!-- ROW END -->
-<br />
+
+##### July 11, 2018
```

Comment:
h5 as a title of h4 doesn't look good but I will approve this.

We can revise these later.